beacon-chain/core/peerdas/validator.go

```diff
@@ -9,7 +9,7 @@ import (
 
 // ValidatorsCustodyRequirement returns the number of custody groups regarding the validator indices attached to the beacon node.
 // https://github.com/ethereum/consensus-specs/blob/v1.5.0-beta.5/specs/fulu/validator.md#validator-custody
-func ValidatorsCustodyRequirement(state beaconState.ReadOnlyBeaconState, validatorsIndex map[primitives.ValidatorIndex]bool) (uint64, error) {
+func ValidatorsCustodyRequirement(state beaconState.BeaconState, validatorsIndex map[primitives.ValidatorIndex]bool) (uint64, error) {
```

Comment:
why was this switch needed from Readonly?

Reply:
No need actually.
Fixed in 3c888de93bc9beb83f918532129b604e8ce05146.